packages/angular/src/directives/control-value-accessors/radio-value-accessor.ts

```diff
@@ -17,13 +17,4 @@ export class RadioValueAccessorDirective extends ValueAccessor {
   constructor(injector: Injector, el: ElementRef) {
     super(injector, el);
   }
-
```

Comment:
Can you explain why you decided to remove the event listener vs. removing the value accessor implementation (directive) completely?

This change still allows `ion-radio` to be used as a form control, but it will never update a bound form control's value after the user selects it.

With the internal ticket, I believe the value accessor implementation for `ion-radio` can be removed completely, as long as we did not observe any issues with doing so. Developers should be binding their form controls to `ion-radio-group`, not `ion-radio`.

Reply:
Should the explanation be placed within the code or the PR description?

Reply:
Discussed via Slack